Userland/Libraries/LibWeb/Page/EventHandler.cpp

```diff
@@ -192,7 +190,7 @@ bool EventHandler::handle_mousewheel(CSSPixelPoint position, unsigned button, un
             if (node->dispatch_event(UIEvents::WheelEvent::create_from_platform_event(node->realm(), UIEvents::EventNames::wheel, offset.x(), offset.y(), position.x(), position.y(), wheel_delta_x, wheel_delta_y, buttons, button).release_value_but_fixme_should_propagate_errors())) {
                 if (auto* page = m_browsing_context->page()) {
                     if (m_browsing_context == &page->top_level_browsing_context())
-                        page->client().page_did_request_scroll(wheel_delta_x * scroll_step_size, wheel_delta_y * scroll_step_size);
+                        page->client().page_did_request_scroll(wheel_delta_x, wheel_delta_y);
```

Comment:
Removing the step size here will have broken scrolling for Browser inside SerenityOS.

Reply:
You are totally right, for now I have moved the `* 24` to `OutOfProcessWebView.cpp`. In the future this can be changed to the system scroll step size setting for example.